Looks like the Linux CI doesn't have flake8 so python-lint tests never run. @shahmishal is it possible to install flake8 on the Linux CI machines for parity with OSX?

However, OSX does correctly identify linting errors:
```
Script:
--
/usr/bin/python /Users/buildnode/jenkins/workspace/swift-PR-osx-smoke-test/branch-master/swift/utils/python-lint
--
Exit Code: 1

Command Output (stdout):
--
./utils/swift_build_support/tests/test_workspace.py:53:80: E501 line too long (83 > 79 characters)
./utils/swift_build_support/tests/test_workspace.py:142:80: E501 line too long (85 > 79 characters)

--
```

--- a/utils/swift_build_support/tests/test_workspace.py
+++ b/utils/swift_build_support/tests/test_workspace.py
@@ -50,7 +50,8 @@ class WorkspaceTestCase(unittest.TestCase):
 
 class ComputeBuildSubdirTestCase(unittest.TestCase):
 
-    def create_basic_args(self, generator, variant, assertions, enable_asan=False):
+    def create_basic_args(self, generator, variant, assertions,
+                          enable_asan=False):
         return argparse.Namespace(
             cmake_generator=generator,
             cmark_build_variant=variant,
@@ -139,7 +140,8 @@ class ComputeBuildSubdirTestCase(unittest.TestCase):
 
         def generate():
             for c in productions:
-                args = argparse.Namespace(cmake_generator="Ninja", enable_asan=False)
+                args = argparse.Namespace(cmake_generator="Ninja",
+                                          enable_asan=False)
                 for key, val in zip(keys, c):
                     setattr(args, key, val)
                 yield compute_build_subdir(args)

